modules/programs/zoxide.nix

```diff
@@ -86,7 +86,7 @@ in {
         if not ($zoxide_cache | path exists) {
           mkdir $zoxide_cache
         }
-        ${cfg.package}/bin/zoxide init nushell ${cfgOptions} | save --force ${config.xdg.cacheHome}/zoxide/init.nu
+        ${cfg.package}/bin/zoxide init nushell ${cfgOptions} | str replace --string --all 'let-env ' '$env.' | save --force ${config.xdg.cacheHome}/zoxide/init.nu
```

Comment:
I'd rather this be fixed in nixpkgs by applying https://github.com/ajeetdsouza/zoxide/pull/587